test/models/EmploymentBenefitsModelSpec.scala

```diff
@@ -34,12 +34,15 @@ class EmploymentBenefitsModelSpec extends UnitTest {
       )
 
   val benefitsViewModel: BenefitsViewModel = BenefitsViewModel(
+    Some(AccommodationRelocationModel(
```

Comment:
same here - can you move the accommodationRelocationModel so it's after the carVanFuelModel